plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/data/LayoutMetaDataService.java

```diff
@@ -51,7 +51,7 @@ private LayoutMetaDataService() {
      * @return  the singleton instance
      */
     public static synchronized LayoutMetaDataService getInstance() {
-        return getInstance(null);
+        return getInstance(LayoutMetaDataService.class.getClassLoader());
```

Comment:
```suggestion
        ObjectLoader loader = null;
        // elkjs-exclude-start
        loader = LayoutMetaDataService.class.getClassLoader();
        // elkjs-exclude-end
        return getInstance(loader);
```

Reply:
I creates this on my phone so please fix potential errors or I will come back to this next year.